video_creation/final_video.py

```diff
@@ -99,8 +99,15 @@ def prepare_background(reddit_id: str, W: int, H: int) -> str:
     try:
         output.run(quiet=True)
     except Exception as e:
-        print(e)
-        exit()
+        print('An error occurred while running...')
+        print('trying auto error fixing...')
+        try:
+            os.remove(f"assets/temp/{reddit_id}/background.mp4")
+            os.remove(f"assets/temp/{reddit_id}/background_noaudio.mp4")
+            prepare_background(reddit_id, W, H)
```

Comment:
This won't work, you first need to get the background video from a different file

Reply:
it does work tho

Reply:
You first need to call `chop_background_video()`, otherwise there is no background

Reply:
one sec going to add it

Reply:
so i did some test and figured out that it works without the chop_background_video() try it out for yourself